cypress/e2e/API/api_EngageSphere.cy.js

```diff
@@ -0,0 +1,160 @@
+        cy.request('GET', CUSTOMERS_API_URL + '/customers?page=1')
+            .then(({ body, status }) => {
+                expect(status, 'Status code').to.be.eq(200);
+
+                body.customers.forEach((customer) => {
+                    expect(customer, 'Customer id').to.have.property('id').that.is.a('number');
+                    expect(customer, 'Customer name').to.have.property('name').that.is.a('string');
+                    expect(customer, 'Number of employees').to.have.property('employees').that.is.a('number');
+
+                    // Verifique se contactInfo é null ou um objeto
+                    if (customer.contactInfo) {
+                        expect(customer.contactInfo).to.be.an('object');
+                        expect(customer.contactInfo, "Name for contact").to.have.property('name')
+                        expect(customer.contactInfo, 'Email for contact').to.have.property('email').to.be.eq(`${customer.contactInfo.email}`)
+                    }
+
+                    // Verifique se address é null ou um objeto
+                    if (customer.address) {
+                        expect(customer.address).to.be.an('object');
+                        expect(customer.address, "Street").to.have.property('street').that.is.a('string');
+                        expect(customer.address, 'City').to.have.property('city').that.is.a('string');
+                        expect(customer.address).to.have.property('zipCode').that.is.a('string');
+                        expect(customer.address).to.have.property('country').that.is.a('string');
+                        expect(customer.address).to.have.property('state').that.is.a('string');
+                    }
+
+                    expect(customer, 'Company Size').to.have.property('size').that.is.a('string');
+                });
+
+            });
+    })
+    context('FILTER CUSTOMERS BY SIZE ON A SINGLE PAGE', () => {
+
+        //MANEIRA CORRETA PARA VERIFICAR TODOS OS TAMANHOS EM UM ÚNICO BLOCO IT
+        //SUGESTÃO DO PROFESSOR!!
+
+        it('FILTERS CUSTOMERS BY SIZE CORRECTLY', () => {
+            const sizes = ['Small', 'Medium', 'Enterprise', 'Large Enterprise', 'Very Large Enterprise']
+            const limitOfEmployessPerSize = [99, 999, 9999, 49999, 999999] // Assuming that there aren't companies with more than 999999 employees in the database
+
+            sizes.forEach((size, index) => {
+                cy.request('GET', CUSTOMERS_API_URL + `/customers?size=${size}`).as('getSizedCustomers')
+
+                cy.get('@getSizedCustomers')
+                    .its('body.customers')
+                    .each(customer => {
+                        expect(customer.size).to.eq(size)
+                        expect(customer.employees).to.be.lte(limitOfEmployessPerSize[index])
+                    })
+            })
+        })
```

Comment:
```suggestion
        })

```

É recomendado deixar uma linha em branco entre cada bloco `it` para melhorar a legibilidade de código e separar os testes.